apps/desktop/src/index.html

```
@@ -1342,7 +1350,10 @@ <h3 id="modal-title" class="text-xl font-light text-[var(--text-primary)] mb-6">
             <div class="space-y-6">
                 <div id="modal-content-area">
                     <!-- Default Input -->
-                    <input type="text" id="modal-input" placeholder="Name..." data-i18n-placeholder="modalInputPlaceholder" class="form-control form-control-lg">
+                    <div class="input-paste-wrapper">
+                        <input type="text" id="modal-input" placeholder="Name..." aria-labelledby="modal-title" data-i18n-placeholder="modalInputPlaceholder" class="form-control form-control-lg">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)